### DISCUSSION TITLE
Welcome to Move2Kube Discussions!

### DISCUSSION THREAD

**Discussion by ashokponkumar** [konveyor/move2kube#316] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with users and collaborators of Move2Kube. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with Move2Kube.

**Comment by sridhariyengar** [comment]
Ashok , we have a customer migrating application workloads from traditional WebSphere apps running in VMs to WebSphere Liberty apps (part of modernization to newer releases ) containerized on OpenShift (on IBM Cloud and or Azure) --- any ideas if this topic has come up ? any plans to address this pattern.  Client is also evaluating Transformation Advisor for some apps

**Reply by ashokponkumar** [reply]
Hi Sridhar, Yes, this topic has come up.

Here are the possible options:
1. For migration of an app to run on Websphere liberty, Transformation Advisor is best suited, as it does it out of the box and generates the artifacts.
2. Out of the box, Konveyor Move2Kube does not attempt to change the server platform, rather containerize the app in the same server platform. But if the user has tens of apps, which follow a specific pattern, then using Konveyor Move2Kube `customization`, the required transformation can be encoded and automated.
